src/components/views/rooms/RoomBreadcrumbs2.tsx

```diff
@@ -88,16 +87,19 @@ export default class RoomBreadcrumbs2 extends React.PureComponent<IProps, IState
 
     private viewRoom = (room: Room, index: number) => {
         Analytics.trackEvent("Breadcrumbs", "click_node", index);
-        defaultDispatcher.dispatch({action: "view_room", room_id: room.roomId});
+        // If we're rendering the first breadcrumb and this is it no-op
+        if (!this.state.skipFirst && index === 0) {
+            return;
+        } else {
+            defaultDispatcher.dispatch({action: "view_room", room_id: room.roomId});
+        }
     };
 
     public render(): React.ReactElement {
-        // TODO: Decorate crumbs with icons: https://github.com/vector-im/riot-web/issues/14040
-        // TODO: Scrolling: https://github.com/vector-im/riot-web/issues/14040
-        // TODO: Tooltips: https://github.com/vector-im/riot-web/issues/14040
         const tiles = BreadcrumbsStore.instance.rooms.map((r, i) => {
             const roomTags = RoomListStore.instance.getTagsForRoom(r);
             const roomTag = roomTags.includes(DefaultTagID.DM) ? DefaultTagID.DM : roomTags[0];
+            const anon = () => this.viewRoom(r, i);
```

Comment:
left over?

Reply:
Oh yeah. My watcher stopped watching and I went crazy trying to figure out why the breadcrumbs worked even when I explicitly broke them. There were many tests before I realised what was going on